apps/passport-server/src/apis/pretix/genericPretixAPI.ts

```diff
@@ -569,6 +584,7 @@ const GenericPretixEventSchema = z.object({
   name: GenericPretixI18MapSchema
 });
 
+// TODO: @rob @richard @josh - can we do a pretix settings scan?
```

Comment:
I believe we produced a guide for event organizers which should explain how to configure Pretix correctly